webpack.config.js

```diff
@@ -4,8 +4,9 @@ const path = require('path')
 
 const webpack = require('webpack')
 
-// Set this to false for easier debugging:
-const production = true
+// DO NOT EDIT THIS CONSTANT
+// Use `yarn prepare.dev` to have EDGE_MODE set appropriately
```

Comment:
As in the core review, this would be better as `// Use "yarn prepare.dev" to make a debug-friendly build:`